bindata/assets/config/defaultconfig.yaml

```diff
@@ -75,6 +75,7 @@ apiServerArguments:
     - StorageObjectInUseProtection
     - TaintNodesByCondition
     - ValidatingAdmissionWebhook
+    - ValidatingAdmissionPolicy
```

Comment:
The plugin is a no-op unless its feature gate is disabled. Adding it here so that we can enable it for tech preview, which was the motivating use case for this PR.